src/main/java/com/gachigang/ontherun/persistence/entity/Token.java

```diff
@@ -18,7 +18,7 @@ public class Token {
     @GeneratedValue
     public Integer id;
 
-    @Column(unique = true)
+    @Column(unique = true, length = 300)
```

Comment:
I added 'length=300' because the token size is bigger than the default value of 255.